Fix build break on VS2015 update 1 RTW

With update 1 RTW, the build breaks. It seems `Nuproj.common` in Windows.Core's project.json file no longer propagates. This may be a bug in Update 1 RTW, and we should follow up on that as well.

C:\Users\andarno\git\pinvoke\src\Kernel32.Desktop\Kernel32.Desktop.csproj : error MSB4057: The target "_NuProjGetProjectClosure" does not exist in the project.
C:\Users\andarno\git\pinvoke\src\Kernel32\Kernel32.csproj : error MSB4057: The target "_NuProjGetProjectClosure" does not exist in the project.

--- a/src/AdvApi32.Desktop/project.json
+++ b/src/AdvApi32.Desktop/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETFramework,Version=v4.0": { }

--- a/src/AdvApi32/project.json
+++ b/src/AdvApi32/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETPortable,Version=v4.0,Profile=Profile92": { }

--- a/src/BCrypt/project.json
+++ b/src/BCrypt/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": {},
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETPortable,Version=v4.0,Profile=Profile92": {}

--- a/src/DbgHelp/project.json
+++ b/src/DbgHelp/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETPortable,Version=v4.0,Profile=Profile92": { }

--- a/src/Gdi32.Desktop/project.json
+++ b/src/Gdi32.Desktop/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": {},
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETFramework,Version=v4.0": { }

--- a/src/Hid.Desktop/project.json
+++ b/src/Hid.Desktop/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETFramework,Version=v4.0": { }

--- a/src/Hid/project.json
+++ b/src/Hid/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETPortable,Version=v4.0,Profile=Profile92": { }

--- a/src/ImageHlp/project.json
+++ b/src/ImageHlp/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETFramework,Version=v4.0": { }

--- a/src/Kernel32.Desktop/project.json
+++ b/src/Kernel32.Desktop/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETFramework,Version=v4.0": { }

--- a/src/Kernel32/project.json
+++ b/src/Kernel32/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETPortable,Version=v4.0,Profile=Profile92": { }

--- a/src/NCrypt/project.json
+++ b/src/NCrypt/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETPortable,Version=v4.0,Profile=Profile92": { }

--- a/src/Psapi/project.json
+++ b/src/Psapi/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETFramework,Version=v4.0": { }

--- a/src/SetupApi.Desktop/project.json
+++ b/src/SetupApi.Desktop/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": { },
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETFramework,Version=v4.0": { }

--- a/src/User32.Desktop/project.json
+++ b/src/User32.Desktop/project.json
@@ -1,6 +1,7 @@
 ﻿{
   "supports": {},
   "dependencies": {
+    "NuProj.Common": "0.10.10-beta-g13363167e1"
   },
   "frameworks": {
     ".NETFramework,Version=v4.0": { }

